tests/common.py

```diff
@@ -113,8 +113,6 @@ def stop_hass():
 @asyncio.coroutine
 def async_test_home_assistant(loop):
     """Return a Home Assistant object pointing at test config dir."""
-    loop._thread_ident = threading.get_ident()
```

Comment:
This is inside the async thread because it's run inside a coroutine. However, I do realize now that this might actually not be the case because during the fixture we're calling `loop.run_until_complete`, which means that we're actually in a thread that executes the fixture! 

We need to somehow schedule this so that it gets executed on the loop.